.github/workflows/push.yaml

```diff
@@ -0,0 +1,20 @@
+# Copyright 2024, UNSW
+#
+# SPDX-License-Identifier: BSD-2-Clause
+
+# Actions to run on Push and Pull Request
+name: CI
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+
+jobs:
+  check:
+    name: License Check
+    runs-on: ubuntu-latest
+    steps:
+    - uses: seL4/ci-actions/license-check@master
```

Comment:
I'm uncomfortable about using checks from repos in a different Github organisation.  Right now this is probably OK, as seL4 is 'close enough' to us; but it'd be better to have our own au-ts checks in the future.

Reply:
I agree but thought it was okay in this case since it's the Foundation.